bindings/python/benchmark.py

```diff
@@ -143,6 +152,17 @@ def to_arrow(use_large):
     find_arrow_all(c, {}, schema=schema)
 
 
+@bench("arrow-roundtrip")
+def roundtrip_arrow(use_large):
```

Comment:
"roundtrip" to me implies read + write while this benchmark only cares about the write. Let's rename to "conventional":
```
@bench("insert-arrow")
def insert_arrow(use_large):
...
@bench("insert-arrow-conventional")
def insert_arrow_conventional(use_large):
```